docs/README-EDITING.md

```diff
@@ -22,15 +22,24 @@ Each Markdown file can start with the following frontmatter tags:
 | `permalink`    | The page's **unique** URL.                    | If not set, gets generated from the Markdown file name.    |
 | `canonicalUrl` | A canonical URL of the page's latest version. | None (not required)                                        |
 | `metaTitle`    | The page's SEO meta title.                    | None (not required)                                        |
+| `description`  | The page's SEO meta description.              | None (not required)                                        |
 | `tags`         | Tags used by the documentation search engine. | None (not required)                                        |
 
+Each tag value can be overwritten. For multi-frameworked pages, there can be a case when the page title, description, or search tags should be different. To overwrite the values for specific framework, use the same tag names and define them under the framework key. The list of supported framework keys: `react`.
```

Comment:
```suggestion
You can set different frontmatter tags for different framework versions of the page. For example, you can set `metaTitle` to say either `JS data grid` or `React data table`, depending on the framework:

\```yaml
// applies to the JS version of the page
metaTitle: JS data grid

// applies to the React version of the page
react:
  metaTitle: React data table
\```

You can use the following framework keys:
- `react`
```